cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageAdmin.java

```diff
@@ -117,11 +124,28 @@ public boolean apply(Relation relation) {
    * @param startMillis start time period
    * @param endMillis end time period
    * @param levels number of levels to compute lineage for
+   * @param resolution lineage collapse resolution
    * @return lineage for sourceDataset
    */
-  public Lineage computeLineage(final DatasetId sourceDataset, long startMillis, long endMillis, int levels)
+  public Lineage computeLineage(final DatasetId sourceDataset, long startMillis, long endMillis,
+                                int levels, String resolution)
     throws NotFoundException {
```

Comment:
move the `throws` to the end of previous line as `int levels, String resolution) throws NotFoundException`